src/main/java/competition/IdealElectricalContract.java

```diff
@@ -81,38 +75,39 @@ public DeviceInfo turretEncoder() {
         return new DeviceInfo(0, true);
     }
 
-    public DeviceInfo carouselMotor()
-    {
+    public DeviceInfo carouselMotor() {
         return new DeviceInfo(35, false);
     }
 
-    public boolean isCarouselReady(){
+    public boolean isCarouselReady() {
         return true;
     }
 
-    public boolean isCollectorArmLiftingReady() {
+    public boolean isArmReady() {
         return true;
     }
 
-    public DeviceInfo liftingCollectorArmMotor() {
-        return new DeviceInfo(5, true);
+    public DeviceInfo armMotor() {
+        return new DeviceInfo(4, true);
     }
 
     public boolean isFrontCollectingReady() {
         return true;
     }
 
     public DeviceInfo frontCollectingMotor() {
-        return new DeviceInfo(6, true);
+        return new DeviceInfo(5, true);
     }
 
     public boolean isHangingReady() {
         return true;
     }
 
     public DeviceInfo hangerMotor() {
-        return new DeviceInfo(7, true);
+        return new DeviceInfo(6, true);
```

Comment:
revert changing these channels, they should stay matching the electrical contract doc